plugins/send2ebook.koplugin/main.lua

```diff
@@ -0,0 +1,229 @@
+        sub_item_table = {
+            {
+                text = _("Download and remove from server"),
+                callback = function()
+                  if not NetworkMgr:isOnline() then
+                      wifi_enabled_before_action = false
+                      NetworkMgr:beforeWifiAction(self.process)
+                  else
+                      self:process()
+                  end
+                end,
+            },
+            {
+                text = _("Go to download folder"),
+                callback = function()
+                    local FileManager = require("apps/filemanager/filemanager")
+                    if FileManager.instance then
+                        FileManager.instance:reinit(download_dir_path)
+                    else
+                        FileManager:showFiles(download_dir_path)
+                    end
+                end,
+            },
+            {
+                text = _("Remove read (opened) articles"),
+                callback = self.removeReadActicles,
+            },
+            {
+                text = _("Set custom download directory"),
+                callback =  self.setCustomDownloadDirectory,
+            },
+            {
+                text = _("Settings"),
+                callback = self.editFtpConnection,
+            },
+            {
+                text = _("Help"),
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = T(_('Send2Ebook lets you send articles found on PC/Android phone to your Ebook reader (using ftp server).\n\nMore details: https://github.com/mwoz123/send2ebook\n\nDownloads to local folder: %1'), download_dir_path)
+                    })
+                end,
+            },
+        },
+    }
+end
+
+function Send2Ebook:lazyInitialization()
+   if not initialized then
+        logger.dbg("Send2Ebook: obtaining download folder")
+        send2ebook_settings = LuaSettings:open(("%s/%s"):format(DataStorage:getSettingsDir(), send2ebook_config_file))
+        if send2ebook_settings:has(config_key_custom_dl_dir) then
+            download_dir_path = send2ebook_settings:readSetting(config_key_custom_dl_dir)
+        else
+            download_dir_path = ("%s/%s/"):format(DataStorage:getFullDataDir(), default_download_dir_name)
+        end
+
+        if not lfs.attributes(download_dir_path, "mode") then
+            logger.dbg("Send2Ebook: Creating initial directory")
+            lfs.mkdir(download_dir_path)
+        end
+    end
+end
+
+function Send2Ebook:process()
+    local info = InfoMessage:new{ text = _("Connecting …") }
+    UIManager:show(info)
+    logger.dbg("Send2Ebook: force repaint due to upcoming blocking calls")
+    UIManager:forceRePaint()
+    UIManager:close(info)
+
+    local count = 1
+    local ftp_config = send2ebook_settings:readSetting("ftp_config") or {address='ftp://Please_setup_ftp_settings_first;)', username="", password="", folder=""}
```

Comment:
Is `ftp://Please_setup_ftp_settings_first;)` a typo of some sort? (Particularly that `;)`)

Also double quotes would be nice. ;-)